.github/workflows/ci.yml

```diff
@@ -147,9 +147,10 @@ jobs:
   unit_tests:
     name: Run Unit Tests
     needs: [build]
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        os: [ubuntu-latest, windows-latest]
```

Comment:
Actually, hmm... On second thought I'm not confident we really need Windows on both Node 16 and Node 20... my gut is that there won't be any significant changes between the two. And we already have quite a big list of CI checks.

Will wait for another @typescript-eslint/triage-team member to chime in.